.github/workflows/ci.yml

```diff
@@ -1265,3 +1265,31 @@ jobs:
         echo "CPPINTEROP_DIR=$CPPINTEROP_DIR" >> $GITHUB_ENV
         echo "LLVM_BUILD_DIR=$LLVM_BUILD_DIR" >> $GITHUB_ENV
         echo "CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH" >> $GITHUB_ENV
+        echo "PREFIX=$PREFIX" >> $GITHUB_ENV
+
+    - name: Build xeus-cpp
+        shell: bash -l {0}
+        run: |
+          micromamba activate CppInterOp-wasm  
+          git clone --depth=1 https://github.com/alexander-penev/xeus-cpp.git
```

Comment:
```suggestion
          git clone --depth=1 https://github.com/compiler-research/xeus-cpp.git
```

Reply:
Apologies, ignore. I do not understand the intent of this pr. I'll sit and wait until is ready.

Reply:
I had already planned to make that change. The reason it currently points to that repo is because that is what is where the PR is which I am trying to help pass is. It will be changed once I get this xeus-cpp passing with wasm.